disk/test/com/treode/disk/DiskTestTools.scala

```diff
@@ -40,6 +41,17 @@ private object DiskTestTools {
 
   val sysid = SystemId (0, 0)
 
+  def assertEqNotification (msgs: String*) (notif: Notification) {
+    notif match {
+      case NoErrors () => assert (msgs.length == 0)
```

Comment:
With the abstract functions, you no longer need to pattern match here. You can just say `msgs == notify.messages`. That kind of weirdness, though, shows why `expected` and `actual` make good param names. It'd be `expected == actual.messages`